src/sentry/static/sentry/app/components/forms/selectControl.jsx

```diff
@@ -87,24 +87,30 @@ const defaultStyles = {
     ...provided,
     lineHeight: '1.5',
     fontSize: theme.fontSizeMedium,
-    color: state.isSelected || state.isFocused ? theme.white : theme.textColor,
-    backgroundColor: state.isSelected
-      ? theme.purpleLightest
-      : state.isFocused
-      ? theme.purpleLight
-      : 'none',
+    color: state.isFocused
+      ? theme.textColor
```

Comment:
As I mentioned before, I’m not a big fan of nested ternaries ..isn't there a better way to accomplish the same? it's just a suggestion 😉

Reply:
I think there are times when nested ternaries can make things more complex, this seems less so since the conditions are pretty simple and the values we return are also simple. We'd be adding 10-15 LOC for returning CSS props.